macros/edr/tests/test_all_columns_anomalies.sql

```diff
@@ -16,20 +16,25 @@
                                                                                    identifier=temp_metrics_table_name,
                                                                                    type='table') -%}
 
-        {#- get column configuration -#}
-        {%- set table_config = elementary.get_table_config_from_graph(model) %}
-        {{- elementary.debug_log('table config - ' ~ table_config) }}
+        {#- get all columns configuration -#}
         {%- set full_table_name = elementary.relation_to_full_name(model) %}
         {%- set model_relation = dbt.load_relation(model) %}
         {%- if not model_relation %}
             {{- elementary.test_log('monitored_table_not_found', full_table_name) }}
             {{- return(elementary.no_results_query()) }}
         {%- endif %}
 
-        {%- set timestamp_column = elementary.insensitive_get_dict_value(table_config, 'timestamp_column') %}
-        {{- elementary.debug_log('timestamp_column - ' ~ timestamp_column) }}
-        {%- set timestamp_column_data_type = elementary.insensitive_get_dict_value(table_config, 'timestamp_column_data_type') %}
-        {{- elementary.debug_log('timestamp_column_data_type - ' ~ timestamp_column_data_type) }}
+        {% if timestamp_column %}
+            {%- set timestamp_column_data_type = elementary.find_normalized_data_type_for_column(model, 'timestamp_column') %}
```

Comment:
The macro 'find_normalized_data_type_for_column' should receive the actual value of the variable timestamp_column and not the string 'timestamp_column' :)